tests/runtime/map

```diff
@@ -19,6 +19,10 @@ NAME map indexed, scalar-like, with decl
 PROG let @a = hash(1); BEGIN { @a[0] = 0; exit(); }
 EXPECT @a[0]: 0
 
+NAME uint64 key
+PROG BEGIN { @a[10223372036854775807] = 0; exit(); }
```

Comment:
just use `UINT64_MAX` (aka 18446744073709551615) to make it completely clear it's a u64?

Reply:
`UINT64_MAX` isn't available in our runtime framework as a const

Reply:
Right, I was just saying if you're gonna use a huge number as a magic value, might as well use UINT64_MAX to make it obvious it's a uint64